### PR TITLE
tauri wallet front-end

### PR DESCRIPTION
**Pull request by fmtabbara**
I merged develop into my branch recently which may be the reason for some of these file diffs?

### CODE REVIEW

**Comment by mmsinclair** [comment]
nymtech/product-team#10

**Review by durch — APPROVED** [review]
I've tested the merge locally it looks good

File: tauri-wallet/src/routes/delegate/DelegateForm.tsx [hunk 1]
```diff
@@ -58,14 +57,22 @@ export const DelegateForm = ({
 
   const onSubmit = async (data: TDelegateForm) => {
     const amount = await majorToMinor(data.amount)
-
-    await invoke(`delegate_to_${data.nodeType}`, {
+    console.log({
```
Comment (mmsinclair) [inline tauri-wallet/src/routes/delegate/DelegateForm.tsx]: Stray console logs?

File: tauri-wallet/src/hooks/useGetBalance.tsx [hunk 2]
```diff
@@ -11,17 +11,17 @@ export type TUseGetBalance = {
 
 export const useGetBalance = (): TUseGetBalance => {
   const [balance, setBalance] = useState<Balance>()
-  const [error, setEror] = useState<string>()
+  const [error, setErorr] = useState<string>()
```
Comment (mmsinclair) [inline tauri-wallet/src/hooks/useGetBalance.tsx]: `setError`? 😂

File: tauri-wallet/src/hooks/useGetBalance.tsx [hunk 3]
```diff
@@ -11,17 +11,17 @@ export type TUseGetBalance = {
...
   const fetchBalance = () => {
     setIsLoading(true)
-    setEror(undefined)
+    setErorr(undefined)
     invoke('get_balance')
       .then((balance) => {
         setBalance(balance as Balance)
       })
-      .catch((e) => setEror(e))
+      .catch((e) => setErorr(e))
```
Comment (mmsinclair) [inline tauri-wallet/src/hooks/useGetBalance.tsx]:
```suggestion
      .catch(setError)
```

Comment (mmsinclair) [reply tauri-wallet/src/hooks/useGetBalance.tsx]: or maybe (unsure about the structure of `e`):

```suggestion
      .catch((e) => setError(e.message))
```

File: tauri-wallet/src/routes/sign-in.tsx [hunk 4]
```diff
@@ -73,7 +73,9 @@ const SignInContent = ({
 }: {
   showCreateAccount: () => void
 }) => {
-  const [mnemonic, setMnemonic] = useState<string>('')
+  const [mnemonic, setMnemonic] = useState<string>(
+    'alley mutual arrange escape army vacuum cherry ozone frame steel current smile dad subject primary foster lazy want perfect fury general eye cannon motor'
```
Comment (mmsinclair) [inline tauri-wallet/src/routes/sign-in.tsx]: Is this an example, or something real? And should it be here?

**Review by mmsinclair — APPROVED** [review]
Looks good - only a few minor things that could be fixed later

**Pull request merged**